Watchman.Engine/AlarmDefinition.cs

```diff
@@ -38,6 +38,11 @@ public class AlarmDefinition
         /// </summary>
         public bool AlertOnOk { get; set; } = true;
 
+        // <summary>
+        // Sets how this alarm is to handle missing data points.
+        // </summary>
+        public string TreatMissingData { get; set; } = "missing";
```

Comment:
could the default for this reference the constant?

Reply:
Yep. Not sure why my linting was complaining when I initially tried to do this. Will add it in.